src/main/fc/cli.c

```diff
@@ -2762,11 +2762,11 @@ static void cliSave(char *cmdline)
 
 static void cliDefaults(char *cmdline)
 {
-    UNUSED(cmdline);
-
     cliPrintHashLine("resetting to defaults");
-    resetEEPROM();
-    cliReboot();
+    resetConfigs();
+    if (isEmpty(cmdline)) {
```

Comment:
Please do a string comparison for the correct argument here. Having random extra characters trigger a different behaviour is wrong.

Reply:
Its done